config/management/genesis/src/storage_helper.rs

```diff
@@ -58,12 +56,9 @@ impl StorageHelper {
 
         // Initialize all other data in storage
         storage
-            .set(
-                SAFETY_DATA,
-                Value::SafetyData(SafetyData::new(0, 0, 0, None)),
-            )
+            .set(SAFETY_DATA, SafetyData::new(0, 0, 0, None))
             .unwrap();
-        storage.set(WAYPOINT, Value::String("".into())).unwrap();
+        storage.set(WAYPOINT, Waypoint::default()).unwrap();
```

Comment:
That was weird `"".into()`

Reply:
to_string <=> into... not sure which one is better but the latter has fewer characters :D